src/modules/core/components/Dialog/Dialog.css

```diff
@@ -8,7 +8,7 @@
 
 .main {
```

Comment:
Please increase the distance between the actual modal and the `circle-close` icon

![Screenshot from 2020-12-10 21-00-21](https://user-images.githubusercontent.com/1193222/101817315-e1ecf480-3b2a-11eb-8ea1-2afcc398112e.png)